mods/common/languages/en.ftl

```diff
@@ -84,6 +84,12 @@ notification-requires-authentication = Server requires players to have an OpenRA
 notification-no-permission-to-join = You do not have permission to join this server.
 notification-slot-closed = Your slot was closed by the host.
 
+## UnitOrders
+notification-game-has-started = The game has started.
+notification-game-saved = Game saved.
+notification-game-paused = The game is paused by { $player }
+notification-game-unpaused = The game is un-paused by { $player }
+
```

Comment:
I think we should add these to `common/languages/rules/en.ftl`

Reply:
Why should this be part of rules, aren't those part of the UI?

Reply:
Yeah they are. Mb